modules/videoio/src/cap_gstreamer.cpp

```diff
@@ -1523,17 +1523,6 @@ bool CvVideoWriter_GStreamer::open( const char * filename, int fourcc,
 
     if (manualpipeline)
     {
-        if (err)
```

Comment:
This code is at best unreachable (if we reach this line, it means that `manualpipeline` is true, and hence it means that `gst_parse_launch` above did not fail), and at worst it triggers an error because `gst_parse_launch` does not guarantee `err` to be NULL if it succeeds.

Reply:
Our "bad" tests are fails and hangs with that change.